src/index.html

```diff
@@ -11,6 +11,48 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+    class="bigSearch"
```

Comment:
Fix classes according to the checklist <img width="1156" alt="Screenshot 2022-12-07 at 16 03 45" src="https://user-images.githubusercontent.com/104769168/206199447-23ca7c74-7dd9-4546-963a-3ae9f622c57a.png">